### PR TITLE
The nextweb

### PR DESCRIPTION
**Pull request by che30**
## The next web project 
In this project we did a clone of the nextweb using : 
- Grids
- Flexbox
- Floats
- Media queries

### CODE REVIEW

File: assets/stylesheets/main.css [hunk 1]
```diff
@@ -0,0 +1,879 @@
...
+.voip {
+  text-align: left;
+  padding: 20px 10px;
+  line-height: 1.5;
+  background-color: #fffaf8;
+  font-size: 14px;
+}
+
+.internet {
+  /* border-bottom: 1px solid grey; */
```
Comment (ghost) [inline assets/stylesheets/main.css:447]: Remove the comment on line 396.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

### Hello team,

**I really enjoyed reviewing your project. Great job guys.**

**Please make the following changes so your project can be approved.**

1). **Your mobile screen size should start at 767px width and not 768px width.**

2). **In the latest news section, for desktop screen size:**

- **capitalize the header.**

- **The image captions are not well spaced out like in the original page.**

latest_clone

3). **Latest funding rounds on index section, at 1024px width**

- **You notice it shows 3 cards instead of four.**

- **The small headings on the rightmost part should display below the large heading.**

- **The content at the top of each card is overlapping each other**

original | clone
---------| -------
funding_round_original | funding_round_clone

4). **Categories (Apps, Gear, Tech, etc) section, at 1024px width.**

- **The contents in the background image and that below the background image overlap each other.**

hardfork_clone

5). **Categories (Apps, Gear, Tech, etc) section, medium screen(768px -1023px).**

- **There should be two sections of content per row and not one.**

original | clone
---------| -------
gear_original | gear_clone


6). **Categories (Apps, Gear, Tech, etc) section, for mobile screen(767px and below).**

- **The text inside the background image should display at the bottom of the background image.**

original | clone
---------| -------
app_original | apps_clone

7). **Add a Getting started section in your readme file and provide details on how to set up your project locally and the necessary tools required. You can use this [guideline](https://github.com/microverseinc/readme-template).**

8). **Wrap the content between the `header` tag and `footer` tag in a `main` tag.**

## Optional
 1). **You can resize the images in the latest news section so they don't appear too stretched. They seem horizontally stretched in large screen sizes and vertically stretched in small screen sizes(767px and below).**

## Happy coding 😃.

**PS: You can reach me on slack (Rose Wanjohi).**

**Review by javitocor — CHANGES_REQUESTED** [review]
# STATUS CHANGES REQUIRED ♻️ 

Good job, team, with the changes implemented! 👏 
However, we still need to do some more changes in order to get the project approved ⏩ 
Kindly check my comments below 👇  apply the changes and resubmit your project for a new code review

## Changes
- Screen size above 1024px:
   - in the latest findings section, the text and the logo are overlapping, kindly fix it
   - HArd fork, apps,... sections, the latest section must have some padding
   - the picture of the latest section in should be 100% height
- Screen size below 768px:
   - there is too much margin between the navbar and the main section, please fix it
   - The third item in the main section must have a caption with the author and the last updated time
   - In the hard fork, apps...sections the text must take almost all the width of the image, please fix it
- General:
   - Try to make the font-size of all the text equal to the original
   - Try to fix the images, the size is not corresponding with the original in the different screens
   - Play a bit more with the padding and margins in all the sections, look at the original and try to make it equal

**Review by Rhelli — APPROVED** [review]
# Approved! 🚀🥳

Hi guys, well done on implementing the changes requested above by the previous TSEs. As you are on your third code review for this project, I thought it best to approve so you can maintain some momentum in your progress through the curriculum. There are however a couple of points I'd like you to take note of though.
 - **[Optional]** - Please ensure you take out all commented out code. There is still commented out CSS in your stylesheets (line 447 👀). Creating titles to separate sections of code is fine but **commented out code should always be removed.**

 - **[Optional]** - I'd have a rethink about how you are approaching your image sizes. Some appear stretched and this is because you are manually setting a width and then the wrong height (or visa-versa). This can be mitigated by setting **only** a width, and then setting the height to `auto` (or visa-versa).

 - **[Optional]** - Double check your font sizes. Some text areas are spilling out of their containers as the font size is too large (i.e. the 'Hard Fork', 'Apps' and 'Gear' section). If you cant get the text to fit, a neat trick is to add the following CSS to the texts container:
`  white-space: nowrap;`
 ` overflow: hidden;`
  `text-overflow: ellipsis;`

(👆🏽 I promise that one will get you out of more than a few jams...)
******
Other than that guys, good work on the project. It's fully responsive and behaves as such, it stays true to theNextWeb's own website structure and your code is nice and neat! Good luck on the next one!

If you have any questions, feel free to @ me here on Github or DM me a message in Slack.

All the best,

💂🏽‍♂️ Rory

**Pull request merged**